liveRecorder/ts_to_mp4_convertor/audio_fillter.c

```diff
@@ -48,9 +76,16 @@ bool generateCodecSilentPacket(AVCodecContext *pContext,AVPacket* packet)
     frame->format         = pNewContext->sample_fmt;
     frame->channel_layout = pNewContext->channel_layout;
     
-    int buffer_size = av_samples_get_buffer_size(NULL, pNewContext->channels, pNewContext->frame_size,pNewContext->sample_fmt, 0);
+    int buffer_size = av_samples_get_buffer_size(NULL, pNewContext->channels, frame->nb_samples ,pNewContext->sample_fmt, 0);
```

Comment:
why did you change parameter from pNewContext->frame_size  to frame->nb_samples ?